jxl/src/frame.rs

```diff
@@ -188,3 +191,39 @@ impl Frame {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod test_frame {
+    use crate::{
+        bit_reader::BitReader,
+        container::ContainerParser,
+        error::Error,
+        headers::{FileHeader, JxlHeader},
+        util::test::assert_almost_eq,
+    };
+
+    use super::Frame;
+
+    fn read_frame(image: &[u8]) -> Result<Frame, Error> {
+        let codestream = ContainerParser::collect_codestream(image).unwrap();
+        let mut br = BitReader::new(&codestream);
+        let file_header = FileHeader::read(&mut br).unwrap();
+        let mut result = Frame::new(&mut br, &file_header)?;
+        result.decode_lf_global(&mut br)?;
+        Ok(result)
+    }
+
+    #[test]
+    fn noise() -> Result<(), Error> {
+        let frame = read_frame(include_bytes!("../resources/test/8x8_noise.jxl"))?;
+        let lf_global = frame.lf_global.unwrap();
+        let noise = lf_global.noise.unwrap();
+        let want_noise = [
```

Comment:
perhaps `want_noise` could be a const, but I'm not sure what is more idiomatic here.

Reply:
then not...